### PR TITLE
Nutanix csi ha zarf init

### LINKED ISSUE #91: Enable HA zarf init
The current configuration of our zarf init is not HA.
We should configure a RWX storage class and scaling parameters for the in-cluster registry as [documented here](https://docs.zarf.dev/ref/init-package/#making-the-registry-highly-available) and in other conversations.

### PR DESCRIPTION
**Pull request by justicorn**
PR to resolve #91

### CODE REVIEW

File: bundles/uds-core-swf/uds-bundle.yaml [hunk 1]
```diff
@@ -15,6 +15,19 @@ packages:
     optionalComponents:
       - git-server
     ref: v0.33.0
+    overrides:
+      zarf-seed-registry:
+        docker-registry:
+          variables:
+            - name: REGISTRY_STORAGE_CLASS
```
Comment (anthonywendt) [inline bundles/uds-core-swf/uds-bundle.yaml:22]: Should these be values or variables? Do they need to be exposed to change at deploy time or set to be configured at create time?

File: bundles/uds-core-swf/uds-bundle.yaml [hunk 2]
```diff
@@ -15,6 +15,19 @@ packages:
...
+              path: persistence.storageClass
+              default: "nutanix-dynamicfile"
+      zarf-registry:
+        docker-registry:
+          variables:
+            - name: REGISTRY_STORAGE_CLASS
```
Comment (anthonywendt) [inline bundles/uds-core-swf/uds-bundle.yaml:28]: Should these be values or variables? Do they need to be exposed to change at deploy time or set to be configured at create time?

File: bundles/uds-core-swf/uds-bundle.yaml (hunk 1 above)
Comment (justicorn) [reply bundles/uds-core-swf/uds-bundle.yaml:22]: For now, we will keep them exposed as variables. To be resolved in [this zarf issue](https://github.com/defenseunicorns/zarf/issues/1725).

File: bundles/uds-core-swf/uds-bundle.yaml (hunk 2 above)
Comment (justicorn) [reply bundles/uds-core-swf/uds-bundle.yaml:28]: For now, we will keep them exposed as variables. To be resolved in [this zarf issue](https://github.com/defenseunicorns/zarf/issues/1725).

**Review by anthonywendt — APPROVED** [review]

**Pull request merged**